services/actions/notifier_helper.go

```diff
@@ -230,16 +230,31 @@ func notify(ctx context.Context, input *notifyInput) error {
 			log.Error("jobparser.Parse: %v", err)
 			continue
 		}
+
+		// cancel running jobs if the event is push
+		if run.Event == webhook_module.HookEventPush {
+			// cancel running jobs of the same workflow
+			if err := actions_model.CancelRunningJobs(
+				ctx,
+				run.ID,
+				run.Ref,
```

Comment:
But isn't that the new one?
I think we want to remove the previous ones?

Reply:
Oh, it should be  ~`run.ID`~`run.RepoID`

Reply:
face5383c7111d3201c03e506ecd92bc85dfd847